account-management-app/src/main/webapp/WEB-INF/jspx/users/user-new.jspx

```diff
@@ -15,9 +15,48 @@
     <tiles:putAttribute
      name="title"
      value="New User Signup" />
-
     <tiles:putAttribute
      name="header-extensions">
+    <c:choose>
+        <c:when test="${recaptchaEnabled}">
+            <script src="https://www.google.com/recaptcha/api.js?render=${recaptchaSiteKey}"><!-- comment --></script>
+            <script>
+                $(function() {
+                (function() {
+                var test = "test";
+                $("input").bind("keypress", function (e) {

```

Comment:
Don't let the user submit the form by hitting the enter key to ensure that recaptcha response is called only when the form is complete.

Reply:
I'm not sure how the indentation go screwed up but that should probably be fixed before merging.